index.js

```diff
@@ -90,6 +90,12 @@ function FSWatcher(_opts) {
     opts.usePolling = process.platform === 'darwin';
   }
 
+  // Global override (useful for end-developers that need to force polling for all
+  // instances of chokidar, regardless of usage/dependency depth)
+  if (process.env.CHOKIDAR_USEPOLLING) {
```

Comment:
`0` coerces to false. Not sure what you're looking for here. It will automatically work for anything that coerces to true, like `1`, any text, the literal word or boolean value `true`